scripts/tests/test_validate_docstrings.py

```diff
@@ -1052,6 +1052,11 @@ def test_raises_for_invalid_attribute_name(self, invalid_name):
         with pytest.raises(AttributeError, match=msg):
             validate_docstrings.Docstring(invalid_name)
 
+    @pytest.mark.parametrize('name', ['pandas.Series.str.isdecimal', 'pandas.Series.str.islower'])
```

Comment:
Hmm I don't think our CI is hitting this directory but this would fail linting for being too long. Can you break lines here after the opening left bracket?

Reply:
Yes, they both fail on windows having characters like '³', '⅕'. I'll break the lines.